test/integration/collection-management/collection_db_management.test.ts

```diff
@@ -16,7 +16,7 @@ describe('Collection Management and Db Management', function () {
   });
 
   it('returns a collection object after calling createCollection', async function () {
-    const collection = await db.createCollection('collection');
+    const collection = await db.createCollection(new ObjectId().toHexString());
```

Comment:
I ran the test suite twice and this failed the second time - I can remove this if we want but this guarantees we never create the same collection twice.